exercises/concept/guidos-gorgeous-lasagna/lasagna_test.py

```diff
@@ -1,11 +1,26 @@
 import unittest
 import pytest
-from lasagna import (
-                    EXPECTED_BAKE_TIME,
-                    bake_time_remaining,
-                    preparation_time_in_minutes,
-                    elapsed_time_in_minutes
-                    )
+import sys
+
+try:
+    from lasagna import (
+                        EXPECTED_BAKE_TIME,
+                        bake_time_remaining,
+                        preparation_time_in_minutes,
+                        elapsed_time_in_minutes
+                         )
+
+
+except ImportError as import_fail:
+    message = import_fail.args[0].split('(')[0]
+    function_name = import_fail.args[0].split()[3][:-1] + "()'"
+
+    if 'EXPECTED_BAKE_TIME' in message:
+        raise ImportError(f"We can't the find the constant 'EXPECTED_BAKE_TIME' in your 'lasagna.py'"
+                          f" file. Did you mis-name or forget to define it?")
```

Comment:
Usually I've seen trailing spaces at the end. I'm not sure if there's a convention around that.